src/libraries/System.Data.Common/src/System/Data/Common/DataAdapter.cs

```diff
@@ -562,6 +571,7 @@ internal int FillFromReader(DataSet? dataset, DataTable? datatable, string? srcT
             return rowsAddedToDataSet;
         }
 
+        [RequiresUnreferencedCode("Parent chapter's type cannot be statically analyzed")]
```

Comment:
I don't see anything about Parent chapter in this method. Can this message be updated to be more obvious about what it means?